### DISCUSSION TITLE
All simplest tutorials for deploying express app to vercel result in Internal Server Error

### DISCUSSION THREAD

**Discussion by DuchSuvaa** [vercel/community#1480] [Help] [answered]
### Summary

I wanted to test Vercel. I need a place where I could host simplest possible express routes for learning purposes.

So I started with simplest tutorials found on the internet, including this one:

https://syntackle.live/blog/how-to-create-and-deploy-an-express-js-app-to-vercel-ljgvGrsCH7ioHsAxuw3G/

I was following them step by step by eventually I always get an Internal Server Error when I want to see the app live.
Let's assume that my code is the code from the link above. Why isn't it working and why am I getting this error?

**Comment by amyegan** [comment, marked as answer]
Hi, @DuchSuvaa! You may be able to find more error details in your [Function Logs](https://vercel.com/docs/concepts/deployments/logs#function-logs). Can you share the example you built? 

I'm not familiar with the tutorial you shared, but we do have a guide on [using Express with Vercel](https://vercel.com/guides/using-express-with-vercel). That one worked for me to deploy an Express site recently

**Comment by DuchSuvaa** [comment]
Time heals wounds. After about 15 minutes the error is gone and it works.
